src/components/LetterGrid/LetterGrid.tsx

```diff
@@ -24,17 +24,13 @@ export const LetterGrid = ({ gameLetters, setWordInput }: LetterGridProps) => {
         }}
       />
       <div className={styles.gridContainer}>
-        {gameLetters.map((letter) => (
-          <div className={styles.flex} key={letter}>
-            <button
-              className={classNames(styles.letter, styles.flex)}
-              value={letter}
-              onClick={handleUserInput}
-            >
-              {letter}
-            </button>
-          </div>
-        ))}
+        {gameLetters.map((letter) =>
```

Comment:
You can make this shorter:

```js
const isPrimary = letter === gameLetters[0]

<LetterBtn variant={isPrimary ? "primary" : "secondary"} key={letter} letter={letter} onClick={handleUserInput} />
```

Reply:
where should the const isPrimary be declared?